packages/pages/src/util/runtime.ts

```diff
@@ -9,22 +11,26 @@ class Runtime {
   version: string;
 
   constructor() {
-    if (typeof process !== "undefined") {
-      this.name = "node";
-      this.version = process.versions.node;
-      this.isServerSide = true;
-    } else if (typeof window !== "undefined" && !("Deno" in window)) {
-      this.name = "browser";
-      this.version = navigator.userAgent;
-      this.isServerSide = false;
-    } else {
+    if (isDeno) {
       this.name = "deno";
       this.version = "";
```

Comment:
Just double checking that it's expected that `version` is a blank string when not in the browser?

Reply:
That was the existing functionality, so it is expected in that sense. I can't speak much more to its reasoning